services/dynamic_addressbook_service.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+option java_package = "com.hederahashgraph.service.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.token">>> This comment is special code for setting PBJ Compiler java package
+
+import "query.proto";
+import "response.proto";
+import "transaction_response.proto";
+import "transaction.proto";
+
+/**
+ * Transactions for Dynamic Address Book Service
+ */
+service DynamicAddressBookService {
```

Comment:
I think we should not introduce a new service but add the new transactions to the NetworkAdmin-service.

Reply:
I checked all rpcs in network_service.proto are queries. In hedera-network-admin-service-impl, we currently include network_service.proto and freeze_service.proto implementation. I think we can add dynamic_addressbook_service.proto implementation in hedera-network-admin-service-impl.

Reply:
I prefer it be in a `node_service.proto`.

Reply:
I put rpc methods in network_service.proto after discussing with @tinker-michaelj and @Neeharika-Sompalli .

Reply:
Changed to node_service.proto according to our discussion on the call today.